Packs/Base/ReleaseNotes/1_8_4.md

```diff
@@ -0,0 +1,4 @@
+
+#### Scripts
+##### CommonServerPython
+- Fixed an issue in **Common**, which caused a failure when running to_context on **FeedRelatedIndicators** list of objects.
```

Comment:
```suggestion
- Fixed an issue in the **Common** class, which would cause a failure while using *to_context* on a list of **FeedRelatedIndicators** objects.
```